xlogger/hooks/deaurorize.go

```diff
@@ -1,4 +1,4 @@
-package xlogger
+package hooks
```

Comment:
cannot do this in minor version bump, or we'd break backward compatibility. Do you suggest a major bump?

Reply:
yes I would suggest a major bump, possibly adding changes to other components of go-dockly as well